content/en/docs/virtualization.md

```diff
@@ -334,3 +334,121 @@ windows.2k22 | Microsoft Windows Server 2022
 windows.2k22.virtio | Microsoft Windows Server 2022 (virtio)
 windows.2k25 | Microsoft Windows Server 2025
 windows.2k25.virtio | Microsoft Windows Server 2025 (virtio)
+
+
+## Virtual Routers
+
+Since version v0.27.0, Cozystack provides a virtual routers functionality.
```

Comment:
A virtual router is usually a cloud-provided entity. Here we provide the ability to use a virtual machine as a router (which usually also requires some additional configuration effort in most clouds). AWS uses the term "middlebox appliance" and GCP uses "router appliance". I'd prefer the latter.